src/components/Table/TableBase.tsx

```diff
@@ -392,11 +392,13 @@ export class TableBase<T> extends React.Component<
 	};
 
 	public toggleChecked = (e: React.ChangeEvent<HTMLInputElement>) => {
+		console.log('*** this.state.sort', this.state.sort);
```

Comment:
Did you mean to keep this debugging line?

Reply:
Nope